src/DIRAC/Core/Tornado/scripts/tornado_start_all.py

```diff
@@ -14,21 +14,24 @@
 
 import os
 import sys
+
+from DIRAC import gConfig
+from DIRAC.ConfigurationSystem.Client import PathFinder
+from DIRAC.ConfigurationSystem.Client.ConfigurationData import gConfigurationData
+from DIRAC.ConfigurationSystem.Client.LocalConfiguration import LocalConfiguration
+from DIRAC.Core.Base import Script
+from DIRAC.Core.Tornado.Server.TornadoServer import TornadoServer
+from DIRAC.Core.Utilities.DErrno import includeExtensionErrors
 from DIRAC.Core.Utilities.DIRACScript import DIRACScript
+from DIRAC.FrameworkSystem.Client.Logger import gLogger
 
 
 @DIRACScript()
 def main():
   # Must be defined BEFORE any dirac import
   os.environ['DIRAC_USE_TORNADO_IOLOOP'] = "True"
 
-  from DIRAC import gConfig
-  from DIRAC.ConfigurationSystem.Client import PathFinder
-  from DIRAC.ConfigurationSystem.Client.ConfigurationData import gConfigurationData
-  from DIRAC.ConfigurationSystem.Client.LocalConfiguration import LocalConfiguration
-  from DIRAC.Core.Tornado.Server.TornadoServer import TornadoServer
-  from DIRAC.Core.Utilities.DErrno import includeExtensionErrors
-  from DIRAC.FrameworkSystem.Client.Logger import gLogger
```

Comment:
Are you sure it's safe to move these? It's a little bit evil but the `DIRAC_USE_TORNADO_IOLOOP` variable is read at import:

https://github.com/DIRACGrid/DIRAC/blob/54d89885089ba29205d50bee9c3bff068e1aed2d/src/DIRAC/FrameworkSystem/Client/MonitoringClient.py#L89-L94

https://github.com/DIRACGrid/DIRAC/blob/8269146d64a8c3345e7c5e785fb70f3a89f6d7b4/src/DIRAC/ConfigurationSystem/private/Refresher.py#L110-L115

Reply:
I was trying to remember why it was in there, and couldn't ! But indeed, they are better inside the main

Reply:
Actually this isn't even working properly at the moment:

```python
In [1]: import sys
   ...: assert "DIRAC.FrameworkSystem.Client.MonitoringClient" not in sys.modules
   ...: assert "DIRAC.ConfigurationSystem.private.Refresher" not in sys.modules
   ...: from DIRAC.Core.Utilities.DIRACScript import DIRACScript
   ...: assert "DIRAC.FrameworkSystem.Client.MonitoringClient" not in sys.modules
   ...: assert "DIRAC.ConfigurationSystem.private.Refresher" not in sys.modules
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-1-8197cf08d195> in <module>
      4 from DIRAC.Core.Utilities.DIRACScript import DIRACScript
      5 assert "DIRAC.FrameworkSystem.Client.MonitoringClient" not in sys.modules
----> 6 assert "DIRAC.ConfigurationSystem.private.Refresher" not in sys.modules

AssertionError:
```